filters/auth/oidc.go

```diff
@@ -618,6 +675,11 @@ func (f *tokenOidcFilter) callbackEndpoint(ctx filters.FilterContext) {
 		}
 	}
 
+	err = f.handleDistributedClaims(r.Context(), oidcIDToken, oauth2Token, claimsMap)
+	if err != nil {
+		log.Error(err)
+	}
+
```

Comment:
Lets move this call into `f.tokenClaims()` as this is a natural extension to it

Reply:
SGTM